packages/gestalt/src/ComboBox.js

```diff
@@ -151,6 +151,12 @@ type Props = {
    * An object representing the zIndex value of the ComboBox list box. Learn more about [zIndex classes](https://gestalt.pinterest.systems/web/zindex_classes)
    */
   zIndex?: Indexable,
+  /**
+   * *EXPERIMENTAL:* Whether to hide ComboBox when reference element gets out of viewport.
```

Comment:
If this prop is experimental, we should add a prefix indicating that. Something like `EXPERIMENTAL_hideWhenReferenceHidden` or similar.

Or we could also make this a hidden prop for now (`_hideWhenReferenceHidden`) and remove the underscore once we deem it no longer experimental.